CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+Add the following to your main PR comment
+=========================
+```
+- [ ] Submitter: Rebase to master. DO NOT SQUASH
+- [ ] Submitter: Make sure Swagger is updated if API changes
+- [ ] Submitter: Make sure documentation for code is complete
+- [ ] Submitter: Remove any TODOs that should be TODOne or create a story for further TODOs
+- [ ] Submitter: Include the JIRA issue number in the PR description
```

Comment:
The current text implies that PRs should have a JIRA issue number, which we want to encourage anyway. I vote leave it as is